stor/dx.py

```diff
@@ -848,6 +848,34 @@ def write_object(self, content, **kwargs):
             suo = OBSUploadObject(fp.name, object_name='/' + self.resource)
             return self.upload([suo], **kwargs)
 
+    def open(self, mode='r', encoding=None):
+        """
+        Opens a OBSFile that can be read or written to and is uploaded to
+        the remote service.
+
+        For examples of reading and writing opened objects, view
+        OBSFile.
+
+        Args:
+            mode (str): The mode of object IO. Currently supports reading
```

Comment:
nit: since this is now using Python 3, can you move type annotations to the method def?